os/arch/arm/src/amebad/Kconfig

```diff
@@ -67,4 +67,17 @@ config AMEBAD_TRUSTZONE
 	default n
 	select ARMV8M_TRUSTZONE
 
+config AMEBAD_RDP
+	bool "Enable Amebad RDP"
+	default n
+	select AMEBAD_RDP_ENABLE
```

Comment:
I could not find AMEBAD_RDP_ENABLE config. Why this config needed?

Reply:
Sorry, this is a mistake, it should be AMEBAD_RDP, i will correct it.